src/components/notifications/Cellsynt.vue

```diff
@@ -0,0 +1,57 @@
+<template>
+    <div class="mb-3">
+        <label for="cellsynt-login" class="form-label">{{ $t("Username") }}</label>
```

Comment:
Are username and password easy to find in their UI => do users need a helptext to know where these are, or are they the general username/passwords?

Reply:
Yes. They send username and password by email and can also be restored on their web site.

Reply:
THEY DO WHAT???

Reply:
![Uploading Screenshot_20240327_231403_Drive.jpg…]()

Reply:
![Screenshot_20240327_231403_Drive](https://github.com/louislam/uptime-kuma/assets/29453612/dafe967f-b2e3-4af2-a2c7-15a9e1c99715)

Reply:
They SEND you your password???

Reply:
After sign-up with details they afterward create an account and send username and passeord by email, yes

Reply:
But they not sending MY passeword to other users ofc. But your point i guess is that email is not Securitas and I agree.

Reply:
Or they create a unique passeword that I can change ... im not sure what answer you looking for?

Reply:
email is not a secure chanel.
=> said pasword has to be assumed to be compromised.

If they do this, I would not be supprised if they store their passwords in plain text.
=> I would not trust such a service with my banking credentials. (but I would also never trust a service which only has a pdf documentation)

Reply:
I don't know how they store the passwords... could be encrypted after generated and sent. They could change the signup steps but nothing I have control over

Reply:
I can forward the issue to the company and see what they answer... im only a customer masking an integration

Reply:
Not sure about banking details. It's invoice I use. So they store my address but dont havet access to my bank 🤔

Reply:
> It's invoice I use

In that case (and if you have changed the password), you should be fine.

Reply:
I got reply from the company. I'm not the best one to forward answers and would attach the answer here if it not was in swedish. 
So they have an understanding about the issue. The problem they have is the choose between greater security and less simplicity for their customers. They have a mix for people with less understanding about the tech and people that use their own web GUI. They send the account details by email and afterward encrypt the passwords. In the mail and when loging in the user get prompt to change the password. 
They also recommend and offer 2FA to make the account more secure.
They have plans to break out the API and they think API-keys is a possible way to go without futher detail explanation.